### PR TITLE
chore: remove unused code (DEV-2950)

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

File: src/dsp_tools/commands/xmlupload/models/value.py [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
 from dsp_tools.models.exceptions import BaseError
-from dsp_tools.models.helpers import Actions, IriTest
-from dsp_tools.models.langstring import LangString
 
 
 @dataclass
-class KnoraStandoffXml:
-    """Used to handle XML strings for standoff markup"""
+class FormattedTextValue:
+    """Represents a formatted text value with standoard standoff markup"""
```
Comment (Nora-Olivia-Ammann) [inline src/dsp_tools/commands/xmlupload/models/value.py:13]: *standard

File: src/dsp_tools/commands/xmlupload/models/value.py [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
 
     xmlstr: str
 
     def as_xml(self) -> str:
+        """Returns the formatted text value as XML (with XML declaration and wrapped in a <text> element)"""
```
Comment (Nora-Olivia-Ammann) [inline src/dsp_tools/commands/xmlupload/models/value.py:18]: What is the advantage to have it as a string as opposed to a etree._Element?

File: src/dsp_tools/commands/xmlupload/models/value.py
Comment (jnussbaum) [inline src/dsp_tools/commands/xmlupload/models/value.py:1]: This file should now be renamed to `formatted_text_value.py`

**Review by jnussbaum — APPROVED** [review]

File: src/dsp_tools/commands/xmlupload/models/value.py (hunk 2 above)
Comment (Nora-Olivia-Ammann) [reply src/dsp_tools/commands/xmlupload/models/value.py:18]: Ok, then it is fine by me leaving it as a string.

File: src/dsp_tools/commands/xmlupload/models/value.py
Comment (BalduinLandolt) [reply src/dsp_tools/commands/xmlupload/models/value.py:1]: ok, then I'll merge it as is

**Pull request merged**